src/screens/IndexScreen.js

```diff
@@ -168,30 +168,26 @@ export const IndexScreen = ({ navigation }) => {
                 }
               });
 
-            let ListHeaderComponent = null;
-
-            if (showFilter) {
-              ListHeaderComponent = (
-                <DropdownHeader {...{ query, queryVariables, data, updateListData }} />
-              );
-            }
-
             return (
-              <ListComponent
-                navigation={navigation}
-                data={listItems}
-                query={query}
-                fetchMoreData={isConnected ? fetchMoreData : null}
-                ListHeaderComponent={ListHeaderComponent}
-                refreshControl={
-                  <RefreshControl
-                    refreshing={refreshing}
-                    onRefresh={() => refresh(refetch)}
-                    colors={[colors.accent]}
-                    tintColor={colors.accent}
-                  />
-                }
-              />
+              <View>
+                {showFilter ? (
+                  <DropdownHeader {...{ query, queryVariables, data, updateListData }} />
+                ) : null}
```

Comment:
to have it a bit shorter and avoid the `: null`, this should work: `{showFilter && <DropdownHeader {...{ query, queryVariables, data, updateListData }} />}`

Reply:
having the `DropdownHeader` not as list header but as component before the list results in different behavior. with that the dropdown is sticky and only the list scrolls. when scrolling the list at the very end, the last item is not visible, because the list has not the full screen space.
this should be fixed.

i think of reverting the change and render the dropdown instead of `null` if there are no list items.

Reply:
another bad side effect is, that the pull to refresh is under the dropdown and the list does not span the whole screen.
if there is just one event, this results in strange scrolling in that small area:

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-01-08 at 12 16 50](https://user-images.githubusercontent.com/1942953/104009498-6f177d80-51ab-11eb-836e-9db9ac5d4126.png)

Reply:
7ac953e72ba1ba13b6f9bb652023493dd73e8a48 and 6ab0a0ddcce8c24cabc377ef520c67fcf10dbcba should fix the issue.

It is not great that cardlist and the other lists behave inconsistently in terms of styling. we should reconsider that. in the future

Reply:
all right, thanks. talk about it tomorrow.